test/runtests.jl

```diff
@@ -1,40 +1,40 @@
 using Test, HTTP, JSON
 
-const dir = joinpath(dirname(pathof(HTTP)), "..", "test")
```

Comment:
can we keep this please? I don't mind moving it down to the for loop, but it's really handy because I often have to step through the test files one by one and `@__DIR__` doesnt' work when run at the REPL.

Reply:
Sure!